### ISSUE TITLE
Joplin 2.6.10 Notebook Icons not synching if sub notebook

### ISSUE THREAD

**Issue by AngelCrawford** [laurent22/joplin#5888]
I updated Joplin on my private PC on the Day of the new Joplin release and immediately added Icons to all my notebooks.
Today I updated Joplin on my work laptop. And the Icons of all Notebooks with sub notebooks are gone missing now.
But Icons in notebooks without an subfolder synched as expected.

![image](https://user-images.githubusercontent.com/16148683/146752219-f5bc7e68-6c71-46a2-aeb7-49f477e64190.png)

"09 - Cosplay" had the same Icon as the sub notebook "Plague Doctor" with the Ghost Icon. 
You can see, that the notebook without a sub folder "19 - Templates" got synched fine, too.

## Environment

Joplin version: 2.6.10
Platform: Windows 10 (Work laptop and Private PC both has the same Platform and Version)

## Steps to reproduce

1.  Add Icons on one PC to a notebook with sub notebooks
2.  Go to another PC and synch everything

## Describe what you expected to happen
I expect all Icons to synch across all PCs and Platforms.

**Comment by AngelCrawford** [comment]
Update: On my Android 12 phone it looks like this. The Icons are synched.

![image](https://user-images.githubusercontent.com/16148683/146753646-684dfdd8-3502-457b-a234-c1bf5fe892c0.png)

**Comment by andre68723** [comment]
+1.

using 2 Macs + Iphone, Subfolder-Icons not visible other than on the original machine created.

Icons started synching once root-notebook got a icon itself

**Comment by seanieb** [comment]
I'm not seeing any of the icons on one of my Macs. 
They're working on my other Mac and iPhone. The Macs are both on the same version of Joplin, MacOS and Syncing from Dropbox.

**Comment by laurent22** [comment]
I cannot replicate this bug with the latest version. For those not seeing any icon, are you sure you are on v2.6?

I think there could also be a problem if you first sync notebooks with icons with 2.5 (in which case icons won't show up), and then upgrade to 2.6. In that case the local data won't have the icons and they won't be downloaded because the app thinks nothing has changed.

To fix this particular case, you can try to make a fake notebook change on the first application (I think if you just edit the notebook and press "OK" it should register a change), then sync this app. Then open the second app and sync again. If you try this, does the icon show up?

**Comment by AngelCrawford** [comment]
You are absolutely right! I think my Work laptop synched with 2.5 first in the Background before I updated to 2.6. 

I tested it right now. You don't have to create a fake notebook. 

Only had to click "Edit" ⇒ "OK" on every notebook in the app where the Icons are, then synch. 
Go to the second app, where the Icons where missing and synch again.
All icons are now synched :)

For me this Issue is solved. Thanks!
